nservicebus/messaging/delayed-delivery.md

```diff
@@ -5,10 +5,21 @@ tags:
 - Defer
 ---
 
-When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this was achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
+When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this could be achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
 
 NOTE: Only send operations can be deferred. You cannot defer a publish or a reply. 
 
+## Difference between versions
+### Defer Handling
+In versions 3-5 by calling `bus.Defer(...)` the handling of the message is deferred by certain amount of time. After that time the message is passed to be handled by the same endpoint, it means that endpoint need to have handler for deferred message to be able to handle it. Using this mechanism delayed delivery can be achieved by calling `bus.Defer(...)` and a having handler in which the actual `bus.Send(...)` will be called after elapsed amount of time. 
```

Comment:
Is this how it works? 

I was under the impression that the `bus.Defer` call would [send the message to the local Timeout Manager](https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Timeout/Core/TimeoutManagerDeferrer.cs#L42). When the Timeout fires [the equivalent of a `bus.Send` occurs](https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Timeout/Core/TimeoutManagerDeferrer.cs#L17).